src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -192,15 +217,15 @@ public String getDaysInLab(String end) {
         return "";
       }
       long daysSinceReceived = Duration.between(receivedByLab.toInstant(), reportingEnd).toDays();
-      if (runs.isEmpty() || !runs.stream().filter(run -> "Running".equals(run.getState())).collect(Collectors.toList()).isEmpty()) {
+      if (runs.isEmpty() || !runs.stream().filter(run -> RUN_RUNNING.equals(run.getStatus())).collect(Collectors.toList()).isEmpty()) {
         return Long.toString(daysSinceReceived);
       } else {
-        RunDto mostRecent = runs.stream().max(Comparator.comparing(RunDto::getCompletionDate)).orElse(null);
+        ModifiedRunDto mostRecent = runs.stream().max(Comparator.comparing(ModifiedRunDto::getCompletionDate)).orElse(null);
         if (mostRecent == null) return Long.toString(daysSinceReceived);
         try {
           runCompleted = getDateTimeFormat().parse(mostRecent.getCompletionDate());
         } catch (java.text.ParseException e) {
-          System.out.println("Error parsing completion date for run " + mostRecent.getId());
+          System.out.println("Error parsing completion date for run " + mostRecent.getRunId());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)